habitat/plan.ps1

```diff
@@ -0,0 +1,127 @@
+$pkg_name="chef-infra-client"
+$pkg_origin="chef"
+$pkg_version=(Get-Content $PLAN_CONTEXT/../VERSION)
+$pkg_upstream_url="https://github.com/chef/chef"
+$pkg_revision="1"
+$pkg_maintainer="The Chef Maintainers <humans@chef.io>"
+$pkg_license=@("Apache-2.0")
+$pkg_filename="${pkg_name}-${pkg_version}.zip"
+$pkg_bin_dirs=@(
+    "bin"
+    "vendor/bin"
+)
+$pkg_deps=@(
+  "core/cacerts"
+  "robbkidd/ruby-plus-devkit"
```

Comment:
you should use `chef/ruby-plus-devkit` ;)

Reply:
![indeed](https://user-images.githubusercontent.com/517302/67028175-e1003480-f0d8-11e9-9237-1f3d8b7c33a5.gif)

dec9dcb